src/app/select/select.sample.html

```diff
@@ -40,9 +40,9 @@ <h4 class="sample-title">Select with ngModel, set items OnInit</h4>
     <div>
         <h4>Display Density</h4>
         <igx-buttongroup (selected)="setDensity($event)">
-            <button igxButton value="compact">Compact</button>
-            <button igxButton value="cosy">Cosy</button>
-            <button igxButton value="comfortable">Comfortable</button>
+            <button igxButton="raised" value="compact">Compact</button>
```

Comment:
You should be able to drop this change safely now. Not that button type does anything in a group, but hey :)